go.mod

```diff
@@ -32,6 +30,52 @@ require (
 	sigs.k8s.io/service-apis v0.1.0
 )
 
+require (
```

Comment:
we have separate require sections for direct and indirect dependency since go 1.17.  Reference can be found here
https://go.dev/ref/mod#:~:text=go%201.14-,require%20directive,-A%20require%20directive